docs/pages/deploy-a-cluster/deployments/gcp.mdx

```diff
@@ -57,18 +41,22 @@ Other things needed:
 Optional:
 
 - Management Tools: Cloud Deployment Manager
-- Stackdriver Logging
+- Logging: Stackdriver
 
-We recommend setting up Teleport in High Availability mode. In High Availability mode Firestore
-stores the state of the system and Google Cloud Storage stores the audit logs.
+We recommend setting up Teleport in high availability mode.
+In high availability mode Firestore is used for cluster state
+and Google Cloud Storage is used for audit logs.
```

Comment:
AFAIK Firestore is used for both cluster state and audit logs - Google Cloud Storage is used for session recordings (i.e. this has always been wrong)

Reply:
Yep, good catch. Thanks G!